### PR TITLE
fix: playwright, and wrong import and export

### PR DESCRIPTION
**Pull request by qperrot**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

wrong import and export !

Also, fixes a playwright test: 

#### Before

When trying to click the 7 day time range, we ended up clicking on dogecoin ! 

Shottr 2024-11-19 11 42 00
Shottr 2024-11-19 11 41 48
SCR-20241119-kjtr

image

SCR-20241119-kjuq

https://github.com/LedgerHQ/ledger-live/actions/runs/11897758228/job/33187953305?pr=8159 

To fix this, we now select within the time range dropdown before clicking on the element containing "7d"

### ❓ Context

- **JIRA or GitHub link**: 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Review by Wozacosta — APPROVED** [review]

File: .changeset/two-apricots-prove.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+"@ledgerhq/coin-stacks": patch
+"ledger-live-desktop": patch
+"@ledgerhq/live-common": patch
```
Comment (Justkant) [inline .changeset/two-apricots-prove.md:4]: You cannot use patch anymore and need to use minor outside hotfix branches

File: libs/ledger-live-common/src/families/stacks/setup.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 import { createBridges } from "@ledgerhq/coin-stacks";
 import stacksResolver, { signMessage } from "@ledgerhq/coin-stacks/signer/index";
-import makeCliTools from "@ledgerhq/coin-xrp/test/cli";
+import makeCliTools from "@ledgerhq/coin-stacks/test/cli";
```
Comment (KVNLS) [inline libs/ledger-live-common/src/families/stacks/setup.ts:5]: Is it intentional?

**Review by KVNLS — APPROVED** [review]
Thanks a lot for this fix!

File: .changeset/two-apricots-prove.md (hunk 1 above)
Comment (Wozacosta) [reply .changeset/two-apricots-prove.md:4]: Oh, already forgot about this morning post, thanks!

File: libs/ledger-live-common/src/families/stacks/setup.ts (hunk 2 above)
Comment (Wozacosta) [reply libs/ledger-live-common/src/families/stacks/setup.ts:5]: No it was a mix up

Comment (qperrot) [reply libs/ledger-live-common/src/families/stacks/setup.ts:5]: yes that was the point of this PR at first then @Wozacosta fixed playwright

File: .changeset/two-apricots-prove.md (hunk 1 above)
Comment (qperrot) [reply .changeset/two-apricots-prove.md]: yes I saw that this morning !
I do the change thx

**Review by ypolishchuk-ledger — APPROVED** [review]

**Pull request merged**